slacksite/slackbot/views.py

```diff
@@ -1,16 +1,57 @@
+from datetime import date, timedelta
 
-from django.http import HttpResponse
+from django.http import JsonResponse
 from django.views.decorators.csrf import csrf_exempt
 
 from db_utils import *
+from models import User
 
 
 @csrf_exempt
 def posting(request):
     r = request.POST
```

Comment:
Probably flake8 is causing issue with the 'import *' part. I can maybe list the modules imported by name to resolve